runtime/meta.c

```diff
@@ -161,32 +162,32 @@ CAMLprim value caml_static_release_bytecode(value bc)
 
 CAMLprim value caml_realloc_global(value size)
 {
+  CAMLparam1(size);
```

Comment:
GC can't be triggered in the code; `Field` no longer includes a read barrier. Make it similar to trunk.

Reply:
Tracking this for post-merge fixing in https://github.com/ocaml/ocaml/issues/10861